flink-formats/flink-orc/src/test/java/org/apache/flink/orc/OrcFileSystemFilterTest.java

```diff
@@ -86,4 +83,49 @@ public void testApplyPredicate() {
         OrcFilters.Predicate predicate8 = new OrcFilters.And(predicate4, predicate6);
         assertThat(predicate7).hasToString(predicate8.toString());
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testApplyPredicateReverse() {
```

Comment:
```suggestion
   void testApplyPredicateReverse() {
```
JUnit5 doesn't need public modifiers, so it's better harden them
could you please also do same for the test method above?

Reply:
Done